### PR TITLE
feat(DialogMatch): add DialogMatch container

### PR DESCRIPTION
**Pull request by zouxuoz**

### CODE REVIEW

File: src/store/reducers.js [hunk 1]
```diff
@@ -10,6 +11,9 @@ const makeRootReducer = (asyncReducers) => {
     auth: authReducer,
     locale: localeReducer,
     user: userReducer,
+    ui: combineReducers({
```
Comment (rosendi) [inline src/store/reducers.js]: Let's add a new line before and after `ui`

File: src/shared/containers/DialogMatch.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  onClose = () => {
+    const { id, closeDialog: close } = this.props;
+
+    close({ id });
+  }
+
+  render() {
+    const { component: Dialog, isOpen, params } = this.props;
+
+    return <Dialog isOpen={isOpen} params={params} onOpen={this.onOpen} onClose={this.onClose} />;
```
Comment (rosendi) [inline src/shared/containers/DialogMatch.js]: `{isOpen && <Dialog...`

**Comment by rosendi** [comment]
lgtm

**Comment by radiansz** [comment]
lgrm

**Pull request merged**